product_docs/docs/biganimal/release/administering_cluster/04_customizing_policy_definitions.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: "Customizing Azure policy definitions"
+description: "Customize default Azure policies to match BigAnimal's resource configurations"
+---
+
+Azure Policies help you monitor, identify, and remediate non-compliant resources. Azure automatically assigns a default set of policies to each subscription. If required by your organization, you can further customize these default Azure policies to match BigAnimal's resource configurations. 
+
+!!! Note
+    BigAnimal does not customize your Azure policies to prevent conflicts with external workloads.
+
+### Customize default policy definitions in Azure
+
+In each of your BigAnimal-enabled Azure subscriptions, manually perform the following steps:
+
+!!! Note
+    You require Microsoft.Authorizations/PolicyAssignments/write permissions to be able to update policy initiatives (sets of policies) in Azure.
+
+
+1. In the Azure portal, enter **Policy** in the search box at the top and open the Policy service.
+
+1. Select **Compliance** on the left side of the Policy page. 
+
+1. On the Compliance page, set the Scope by selecting the ellipsis and then selecting all subscriptions. Click the **Select** button at the bottom of the Scope page to add your selection.
+
+1. You can see a list of all the policy initiatives (sets of policies) automatically assigned by Azure's onboarding process. The policy initiative for each subscription is labeled as *ASC Default (subscription: <Subscription_ID>)*
```

Comment:
Generally would want to escape those angle brackets, but seems good for now

Reply:
I jumped the gun a bit merging that PR. It will go through an editing pass tomorrow and we will address the angle brackets at that time.